test/Objects/Schema/Generation/SchemaGeneratorTest.php

```diff
@@ -156,9 +159,17 @@ public function it_should_generate_records_containing_records()
                     ->name('SimpleRecord')
                     ->namespace('org.acme')
                     ->field('intType', Schema::int(), Schema\Record\FieldOption::default(42))
+            )
+            ->field(
+                'union',
```

Comment:
Nit: Should use the name constant.

Reply:
I don't think so. Here the union name does not represent the union type itself, it is just the name of the field. Maybe the best would be to give the field another name to avoid confusion?

Reply:
The name did indeed confuse me. I like the idea of renaming it to something that indicates an example fieldname rather than the type.